src/CSET/operators/plot.py

```diff
@@ -320,6 +320,43 @@ def _plot_and_save_line_series(
     plt.close(fig)
 
 
+def _plot_and_save_vertical_line_series(
+    cube: iris.cube.Cube, coord: iris.coords.Coord, filename: str, title: str, **kwargs
+):
+    """Plot and save a 1D line series.
+
+    Parameters
+    ----------
+    cube: Cube
+        1 dimensional Cube of the data to plot on x-axis.
+    coord: Coord
+        Coordinate to plot on y-axis.
+    filename: str
+        Filename of the plot to write.
+    title: str
+        Plot title.
+    """
+    fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
+    iplt.plot(cube, coord, "o-")
+    ax = plt.gca()
+    ax.invert_yaxis()
+
```

Comment:
Given the inverted y axis I am assuming this operator is specifically for pressure levels and model levels have not been catered for yet (and I assume would appear in a later issue/PR). Therefore, it would be good to have the y-axis plotted on a log scale rather than a linear scale.

You may then also want to think about defining specific y tick labels as well, given how the log scale produces the tick labels.

Reply:
Thanks for suggestions. I changes the y axis to log pressure and set fixed limits. I also set fixed limits for the x axis based on the entire cube data so that the x axis does not change over the sequence coordinate.
Commit vertical_profiles_of_area_averaged_fields_494 256f3ab